src/forms/FormFieldAdvancedSelection/AdvancedSelectionDrawer.tsx

```diff
@@ -281,6 +292,9 @@ const AdvancedSelectionDrawer = (props: AdvanceSelectionDrawerPropTypes): ReactE
 	 * @param checkedOptions
 	 */
 	const checkboxListChanged = (checkedOptions: MosaicLabelValue[]) => {
+		if (checkedOptions.length >= fieldDef.inputSettings?.selectLimit) {
+			setCheckboxListDisabled(true)

```

Comment:
Please don't forget the ";" at the end of each line. It might not be as important in javascript but in typescript it can lead to production errors.